greedy_algorithms/dijkstra.cpp

```diff
@@ -1,29 +1,38 @@
-#include <limits.h>
+/**
+ * @file
+ * \brief Implementation of Dijkstra's Algorithm
+ * \details
+ * Dijkstra's Algorithm is a greedy solution for the single-source shortest path problem in a graph whose edges have integer weights.
+ * This solution takes in input to build the graph and outputs distances to nodes from a selected source node.
+ * @author ali-mir
```

Comment:
Were you the author of this code?
[file history](https://github.com/TheAlgorithms/C-Plus-Plus/blame/70a2aeedc369fbbc7313b768e0f5e9c673bcd86f/greedy_algorithms/Dijkstra.cpp)
If not, please add the first author and then your name and in the documentation both will be recorded.
```suggestion
 * @author <original author>
 * @author ali-mir
```

Reply:
I've added additional authors who contributed logic to this file. I refrained from adding one of the committers because the changes were a part of a massive, multi-file format commit. Let me know if you would like that name added as well.